inc/setup/class-post-modifier-rest.php

```diff
@@ -17,24 +17,32 @@ public function register_routes() {
 			'callback'            => array( 'post_modifier_Rest', 'al_change_settings' ),
 			'permission_callback' => '__return_true',
 		) );
+
+		register_rest_route( static::$namespace, '/image' . '/(?P<id>[\d]+)', array(
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
+				'callback'            => array( 'post_modifier_Rest', 'al_get_image' ),
+				'permission_callback' => '__return_true',
+			),
+		) );
+	}
+
+	/**
+	 * @param WP_REST_Response $request
+	 */
+	public static function al_get_image( $request ) {
+		Post_Modifier_Settings::getInstance()->updateSetting( 'site_logo_src', wp_get_attachment_image_src( $request->get_param( 'id' ) )[0] );
+		echo( json_encode( get_option( 'site_logo_src' ) ) );
```

Comment:
```suggestion
		return new WP_REST_Response( get_option( 'site_logo_src' ), 200 );
```
I think this should work without json encode